From bbf37b33dd146af3c9b9842a1c6e492cbde42c8c Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Mon, 03 Jan 2022 14:21:30 +0200
Subject: [PATCH] [SW-69129] gaudi2: boot info: update bootinfo base address

This patch move boot info base to 96 KB offset in PID ICCM.
32 KB will be available for boot info,
thereby leaving 96 KB for arc pid application, out of total 128 KB.
With 64 KB limit for arc pid, ICCM was overflowing,
when new code is being added.

Change-Id: I8c8a01256b157bd9ab38548a27a058d0714b94b4
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index 9ca6b24..2f109cd 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -63,7 +63,7 @@
 #define PPBOOT_TEXT_BASE	(FW_SRAM_BASE + PPBOOT_START_OFFSET)
 #define PREBOOT_TEXT_BASE	(SDRAM_BASE + PREBOOT_START_OFFSET)
 
-#define ICCM_BOOT_INFO_OFFSET   0x10000
+#define ICCM_BOOT_INFO_OFFSET   0x18000
 #define BOOT_INFO_BASE          (ARC0_ICCM_BASE + ICCM_BOOT_INFO_OFFSET)
 
 /* In order not to conflict with ARC BOOTROM's usage of DCCM,
diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index 2e0ac15..7fe10cb 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -58,7 +58,7 @@
 
 #define HL_INIT_SP_SRAM_RETRIES		50
 
-#define ICCM_BOOT_INFO_OFFSET		0x10000
+#define ICCM_BOOT_INFO_OFFSET		0x18000
 #define ARC0_ICCM			0x3FF7000000
 #define BOOT_INFO_BASE			(ARC0_ICCM + ICCM_BOOT_INFO_OFFSET)
 
